share/spack/spack-completion.bash

```diff
@@ -59,8 +84,7 @@ _bash_completion_spack () {
     # which is true if the current word starts with '-' or if the cursor is
     # not at the end of the line.
     local list_options=false
-    if [[ "${COMP_WORDS[$COMP_CWORD]}" == -* || \
-          "$COMP_CWORD" -ne "${#COMP_WORDS[@]}-1" ]]
+    if [[ "${COMP_WORDS[$COMP_CWORD]}" == -* || "$COMP_CWORD" -ne "${#COMP_WORDS[@]}-1" ]]
```

Comment:
Now that I think about it, that subtraction in `"${#COMP_WORDS[@]}-1"` definitely doesn't work...

Reply:
If I change it to `$(("${#COMP_WORDS[@]}" - 1))` the code makes more sense, but the logic still isn't working. I swear I had this working when I first wrote it, but it's been 5 years since then, so who knows...

Reply:
Fixed in latest commit